app/api/helpers/tasks.py

```diff
@@ -501,6 +502,9 @@ def export_order_pdf_task(self, event_id):
 @celery.task(base=RequestContextTask, name='export.attendees.csv', bind=True)
 def export_attendees_csv_task(self, event_id):
     attendees = db.session.query(TicketHolder).filter_by(event_id=event_id)
+    custom_forms = db.session.query(CustomForms).filter_by(
```

Comment:
Filter out deleted custom forms

Reply:
This model extends db.Model and not soft deleted model

Reply:
OK